apps/backend/src/datasources/postgres/FapDataSource.ts

```diff
@@ -471,7 +471,8 @@ export default class PostgresFapDataSource implements FapDataSource {
         'p.status_id': 'ps.proposal_status_id',
       })
       .where('sp.fap_id', fapId)
-      .andWhere('ihp.instrument_id', instrumentId);
+      .andWhere('ihp.instrument_id', instrumentId)
+      .andWhere('p.submitted', true);
```

Comment:
I think in the teams thread you mentioned it was pulling in draft proposals and expired ones. I can see how this'll exclude any draft proposals, but is this enough, for current needs at least? Can something be submitted and then marked as expired? Or does expiring unset any submitted flag?

Also, could you put the details of why this needed into an issue so it's easier to look back on why we wanted the change made.